developers_chamber/ecs_utils.py

```diff
@@ -193,25 +193,64 @@ def start_service(cluster, service, count, region):
     except ecs_client.exceptions.ClusterNotFoundException:
         raise ClickException("Cluster not found: '{}'".format(cluster))
     except ecs_client.exceptions.ServiceNotFoundException:
-        raise ClickException("Service not found: '{}'".format(cluster))
+        raise ClickException("Service not found: '{}'".format(service))
     except ClientError as ex:
         raise ClickException(ex)
 
 
-def start_services(cluster, count, region):
+def start_services(cluster, services, count, region):
+    ecs_client = _get_ecs_client(region)
+
+    if count is not None and count < 1:
```

Comment:
Na tohle vždycky křičel Luboš, že stačí `if count ...` - is not None není nutné.

Reply:
```
if count is not None and count < 1:
```
count = None OK
count = 1 OK
count = 0 Raise
count = -1 Raise

```
if not count and count < 1:
```
count = None TypeError (porovnání None < 1)
count = 1 OK
count = 0 Raise
count = -1 OK


```
if not count or count < 1:
```
count = None Raise 
count = 1 OK
count = 0 Raise
count = -1 Raise

Možná mi něco nedochází, ale přijde mi že ta podmínka nejde jinak udělat

None je validní hodnota

Reply:
A co ti vyhodí `if count and count < 1` ?

Reply:
(Samozřejmě by to na jiných místech mělo fungovat i obráceně: `if count is None` jako `if not count`)

Reply:
```if count and count < 1:```
count = None OK
count = 1 = OK
count = 0 = OK <-- chyba
count = -1 = Raise

Reply:
Asi jsem dneska nějaký nedovtipný, na to jsem zapoměl...:(